components/layout/block.rs

```diff
@@ -1491,7 +1491,7 @@ impl BlockFlow {
         debug_assert!(!self.is_inline_flex_item());
 
         // Compute the available space for us, based on the actual floats.
-        let rect = self.base.floats.available_rect(Au(0),
```

Comment:
Whoops, didn't see you comment, Sure I can add a test that utilize both :) I'm not famillar with incremental layout infrastructure, but they are the only flags `AssignISizes` and `AssignBSizes` check, so if they were any problem please comment :)